src/Common/src/Interop/Windows/Mincore/Interop.DynamicLoad.cs

```diff
@@ -14,7 +14,7 @@ internal static unsafe partial class mincore
         [DllImport("api-ms-win-core-libraryloader-l1-2-0.dll")]
         internal static extern IntPtr GetProcAddress(IntPtr hModule, byte* lpProcName);
 
-        [DllImport("api-ms-win-core-libraryloader-l1-2-0.dll", CharSet = CharSet.Unicode)]
```

Comment:
CoreCLR uses UTF8 for the symbol name conversion. This will use ACP. Convert to UTF8 manually instead?